packages/perspective-viewer/src/js/row.js

```diff
@@ -171,6 +176,8 @@ class Row extends HTMLElement {
         let agg_dropdown = this.querySelector('#column_aggregate');
         let aggregate = this.getAttribute('aggregate');
         if (agg_dropdown.value !== aggregate) {
+            // TODO: Should type be removed here? What is it doing
+            // eslint-disable-next-line no-undef
             agg_dropdown.value = aggregate || perspective.AGGREGATE_DEFAULTS[type];

```

Comment:
`type` here seems completely undefined, so I assume that `perspective.AGGREGATE_DEFAULTS[type]` will also return `undefined` in all cases. If so can we just remove the right hand side of the or?

Reply:
Yes, aggregate is always defined now - this looks like a legitimate catch by eslint!

Reply:
+1 for tooling! I'll remove the right hand side and remove the comment.